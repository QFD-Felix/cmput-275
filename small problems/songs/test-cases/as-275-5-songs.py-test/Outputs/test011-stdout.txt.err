5 16 1 12 2 11 1 1 1 15 1 23 0 1 8 5 5 8 0 4 7 19 13 14 4 5 0 2 4 17 1 2 26 14 8 1 2 16 1 17 1 3 1 2 24 3 6 12 2 0 1 3 15 10 1 7 21 0 15 2 0 4 2 7 2 4 6 8 16 9 1 3 7 9 2 14 4 8 3 0 21 10 4 0 3 8 2 3 19 6 10 7 0 1 12 1 3 0 6 4 8 1 0 0 0 0 0 0 0 0
