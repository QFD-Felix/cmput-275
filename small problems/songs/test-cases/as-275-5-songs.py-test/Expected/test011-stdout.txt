5 16 1 12 2 11 1 1 1 1 1 13 0 1 8 5 5 8 0 4 7 19 13 14 4 5 0 2 4 17 1 2 27 14 0 1 2 5 1 10 1 3 1 2 25 3 7 13 3 1 7 1 3 15 0 1 7 7 0 15 2 0 4 2 7 2 4 6 8 18 9 1 3 7 9 2 16 1 10 2 5 5 0 6 0 5 0 17 8 2 3 19 6 10 0 0 1 12 1 3 0 0 4 8 1 0 0 0 0 0 0 0 0
